protected/helpers/dbMaster.php

```diff
@@ -280,7 +280,7 @@ public static function tableExists($name) {
     }
 
     private static function newGetColumns($tableName, $prefix) {
-        $sql = 'SELECT * FROM ' . $tableName . " LIMIT 1";
+        $sql = 'SELECT * FROM `' . $tableName . "` LIMIT 1";
```

Comment:
I changed only 2 lines, 283 abd 294 (the sql statements). I am not sure why other lines are also shown as changed (perhaps due to differences in line endings)